packages/docusaurus-theme-classic/src/theme/Icon/Socials/Github/index.tsx

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import type {SVGProps} from 'react';
+import {useColorMode} from '@docusaurus/theme-common';
+
+// SVG Source: https://svgl.app/
+function Github(props: SVGProps<SVGSVGElement>): JSX.Element {
+  const {colorMode} = useColorMode();
+  const fillColor = colorMode === 'dark' ? '#fff' : '#000';
+  return (
+    <svg
+      viewBox="0 0 256 250"
+      width="256"
+      height="250"
+      {...props}
+      xmlns="http://www.w3.org/2000/svg"
+      fill={fillColor}
+      preserveAspectRatio="xMidYMid">
```

Comment:
TODO myself, investigate "execution order", className is executed after props, and if we reverse the order then the black white css doesn't work anymore from what i've concluded